app.json

```diff
@@ -1,21 +1,8 @@
 {
-  "name"              : "node-app",
-  "script"            : "app.js",
-  "log_date_format"   : "YYYY-MM-DD HH:mm Z",
-  "merge_logs"        : false,
-  "watch"             : false,
-  "max_restarts"      : 10,
-  "env":
-  {
-    "NODE_ENV"        : "production",
-    "BOOTNODES"      : "52.194.200.217,52.78.22.32,18.136.251.28",
-    "BOOTNODE_PORT"  : "32323",
-    "RPC_PORT"        : "8651",
-    "SUB_RPC_PORT"    : "8551",
-    "LISTENING_PORT"  : "30303",
-    "INSTANCE_NAME"   : "klaytn-net-intelligence-api",
-    "WS_SERVER"       : "http://localhost:3000",
-    "WS_SECRET"       : "klaytn-net-stats-has-a-secret",
-    "VERBOSITY"       : 2
-  }
-}
+  "name": "node-app",
```

Comment:
@kjeom Can we change the app name shown on pm2?

Reply:
Sure, klaytn-net-intelligence-api would be good.